install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -131,30 +134,38 @@ spec:
                   custom:
                     properties:
                       url:
+                        description: URL of an already running external Broker instance
                         type: string
                     type: object
                   native:
                     properties:
                       name:
+                        description: Name of the Broker instance to be deployed
                         type: string
                       namespace:
+                        description: Namespace that the Broker instance to be deployed would run in
                         type: string
                     type: object
                 type: object
               version:  
+                description: The current version of the meshery application as from its release
```

Comment:
Please always capitalize Meshery, unless you're referring to a logical item like the `meshery` Namespace.

Reply:
Alright, Noted.